stylesheets/_modules.scss

```diff
@@ -310,6 +310,7 @@
   line-height: 0;
   display: flex;
   flex-direction: column;
+  width: 100%;
```

Comment:
Nitpick: I don't think this is necessary after 84f4e6ac6424c926ba3dc41aa1483babf1dfcce4

Reply:
Yup, you're right. I've removed this change.